store/models/orm.go

```diff
@@ -397,6 +398,13 @@ func (orm *ORM) AuthorizedUserWithSession(sessionID string) (User, error) {
 	if err != nil {
 		return User{}, err
 	}
+	if session.LastUsed.Time.Add(sessionDuration).Before(time.Now()) {
+		return User{}, errors.New("Session has expired")
+	}
+	session.LastUsed = Time{Time: time.Now()}
```

Comment:
Maybe we should store the instance of time.Now() so it is consistent? Really small chance of a bug, but it would have fewer method calls, and the behavior would be consistent between the time passing the check and the last used time in the db.